src/components/UserSelector/UserSelector.tsx

```diff
@@ -0,0 +1,88 @@
+import React, { useEffect, useRef, useState } from 'react';
+import cn from 'classnames';
+import { User } from '../../types/User';
+
+type Props = {
+  users: User[];
+  selectedUserId: number;
```

Comment:
you could fetch users inside this component, you are not using them anywhere else

Reply:
good idea